doc/conf.py

```diff
@@ -61,6 +59,46 @@ def run_doxygen():
         os.chdir(curdir)
 
 
+def build_jvm_docs():
+    """Build docs for the JVM packages"""
+    git_branch = os.getenv("READTHEDOCS_VERSION_NAME", default=None)
+    print(f"git_branch = {git_branch}")
+
+    if git_branch == "latest":
+        git_branch = "master"
+    elif git_branch == "stable":
+        git_branch = f"release_{version}"
+
```

Comment:
```suggestion
    else:
        git_branch = "master"
```
I need to build the doc regularly, a good default would be really helpful.